x/staking/keeper/val_state_change.go

```diff
@@ -192,14 +194,17 @@ func (k Keeper) ApplyAndReturnValidatorSetUpdates(ctx sdk.Context) (updates []ab
 	}
 
 	for _, valAddrBytes := range noLongerBonded {
-		validator := k.mustGetValidator(ctx, sdk.ValAddress(valAddrBytes))
+		valAddr := sdk.ValAddress(valAddrBytes)
+		validator := k.mustGetValidator(ctx, valAddr)
```

Comment:
revert?

Reply:
what do you mean?

Reply:
is this change needed? it looks like an unessecary change

Reply:
Ah yeah, no longer needed. 
```suggestion
		validator := k.mustGetValidator(ctx, sdk.ValAddress(valAddrBytes))
```